src/django/setup.cfg

```diff
@@ -0,0 +1,2 @@
+[flake8]
+max-line-length = 88
```

Comment:
@rachelekm We're going with `pyproject.toml` instead, generally?

Reply:
If we're anticipating integrating GH Super-Linter support into this project we can define `black` and `flake8` config files as environment values that will override the linters defaults, so no restrictions there! I'll defer to @rajadain if there's a project-specific preference here.

Reply:
No project specific preference. Let's go with the standard.

Reply:
@rajadain Over Slack we discussed the possibility of this changing when the Super-Linter is brought into this project, but until then, `echo-locator` for instance used `setup.cfg`: https://github.com/azavea/echo-locator/blob/develop/.github/super-linter.env#L6-L7

Reply:
Sorry, missed your replies before posting.

Reply:
Generally prefer consolidating configuration in one place if we can, rather than having both `pyproject.toml` and `setup.cfg`. If both `flake8` and `black` can be configured in one but not the other, let's go with that. If both can be setup in both, than I have a slight preference for `pyproject.toml` which feels more modern.